src/main/java/org/terasology/structureTemplates/util/ProtectedRegionUtility.java

```diff
@@ -35,12 +34,13 @@
      * @return
      */
     public static boolean isInProtectedRegion(Collection<Vector3i> positions, EntityRef regionEntity) {
-        ProtectedRegionsComponent protectedRegionsComponent = regionEntity.getComponent(ProtectedRegionsComponent.class);
-        List<Region3i> protectedRegions = protectedRegionsComponent.regions;
+        ProtectedRegionsComponent protectedRegionsComponent =
+                regionEntity.getComponent(ProtectedRegionsComponent.class);
+        List<BlockRegion> protectedRegions = protectedRegionsComponent.regions;
         if (protectedRegions != null) {
-            for (Region3i region : protectedRegions) {
+            for (BlockRegion region : protectedRegions) {
                 for (Vector3i position : positions) {
-                    if (region.encompasses(position)) {
+                    if (region.containsPoint(JomlUtil.from(position))) {
```

Comment:
BlockRegion also has a containsBlock function that is non inclusive. double check with encompasses to make sure the implementation is correct.